apps/token-manager/app/src/script.js

```diff
@@ -47,67 +44,66 @@ retryEvery(retry => {
   })
 })
 
-async function initialize(tokenAddr) {
-  const token = app.external(tokenAddr, tokenAbi)
-  try {
-    const tokenSymbol = await token.symbol().toPromise()
-    app.identify(tokenSymbol)
-  } catch (err) {
-    console.error(
-      `Failed to load token symbol for token at ${tokenAddr} due to:`,
-      err
-    )
-  }
-
-  return createStore(token, tokenAddr)
-}
+async function initialize(tokenAddress) {
+  const token = app.external(tokenAddress, tokenAbi)
 
-// Hook up the script as an aragon.js store
-async function createStore(token, tokenAddr) {
   return app.store(
     async (state, { address, event, returnValues }) => {
```

Comment:
We can do this later, but it may be nice to extract the `reducer` out into its own function from now on, now that there's a distinction between the `init` and the `reducer`. It may make it more clear at a glance what the `createStore()` is hooked up to.